src/libraries/System.Private.CoreLib/src/ILLink/ILLink.Substitutions.Shared.xml

```diff
@@ -19,4 +19,13 @@
       <field name="s_asyncDebuggingEnabled" value="false" initialize="false" />
     </type>
   </assembly>
+  <assembly fullname="System.Private.CoreLib" feature="System.Runtime.InteropServices.Marshal.IsComSupported" featurevalue="false">
+    <type fullname="System.Type" >
+      <method signature="System.Boolean get_IsCOMObject()" body="stub" value="false" />
+    </type>
+    <type fullname="System.Runtime.InteropServices.Marshal">
+      <method signature="System.Boolean IsComObject(System.Object)" body="stub" value="false" />
+      <method signature="System.Boolean IsTypeVisibleFromCom(System.Type)" body="stub" value="false" />
+    </type>
+  </assembly>
```

Comment:
The feature switch should only substitute the IsComSupported property.
Everything else should be done in code using the IsComSupported property - so that we get consistent behavior with/without linker.